.golangci.yml

```diff
@@ -181,6 +181,10 @@ issues:
       linters:
         - errcheck
 
+    - text: "don't use an underscore"
```

Comment:
What's this for?

Reply:
package core.oam.dev can only used by `core_oam_dev` while golint complain this message